Dalamud/Plugin/Internal/Loader/PluginLoader.cs

```diff
@@ -151,6 +151,9 @@ private static AssemblyLoadContextBuilder CreateLoadContextBuilder(LoaderConfig
             builder.PreferDefaultLoadContextAssembly(assemblyName);
         }
 
+        builder.PreferDefaultLoadContextAssembly(Assembly.GetExecutingAssembly().GetName());
```

Comment:
Could you maybe add a comment explaining what these do?

Reply:
Added some comments (and swapped the two lines, so it makes more sense)